kernels/optimized/cpu/op_log_softmax.cpp

```diff
@@ -55,7 +55,7 @@ void log_softmax_kernel(const Tensor& input, int64_t dim, Tensor& out) {
         0,
         outer_size,
         ::executorch::extension::internal::GRAIN_SIZE,
-        [&](const auto begin, const auto end) {
+        [&](const auto& begin, const auto& end) {
```

Comment:
not sure I understand why we have to pass ints by reference here. if this doesn't get inlined it will cost a little size